examples/cs/core/simple_json/simple_json.csproj

```diff
@@ -1,66 +1,31 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\..\..\cs\build\nuget\Bond.CSharp.props" />
   <PropertyGroup>
-    <ProjectGuid>{45E4032C-78E1-41dd-8E51-1CC5AD2AEE02}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <RootNamespace>simple_json</RootNamespace>
     <AssemblyName>simple_json</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworks>net45</TargetFrameworks>
+    <IntermediateOutputPath>obj\$(BuildType)\$(BuildFramework)</IntermediateOutputPath>
```

Comment:
Explicitly setting IOP seems odd here. Is this to make the test discovery PowerShell script still work? If so, can we make that script smarter instead of playing games with IOP?

Reply:
I was seeing weird behavior regarding IOP and the codegen target. It seems it is blank when it is running codegen and tries to write into c:\codegen.in file, etc. It's a current issue.

Reply:
There is an issue here when there is no IOP explicitly set

Reply:
Yeah. The order of the targets imports has changed slightly. Your new `SetBondOutputDirectory` target LGTM.